tests/test_io.py

```diff
@@ -93,7 +93,12 @@ def test_write_yaml(tmp_path: Path) -> None:
 
 def test_path_to_str(tmp_path: Path) -> None:
     """Test that Path objects are converted to strings."""
-    CONFIG_PATH = {"this": "is", "a": "test", "with": tmp_path, "and": {"nested": tmp_path / "nested"}}
+    CONFIG_PATH = {
```

Comment:
Curious what linter (`black` or `ruff>=0.1.4`-ish) have decided to wrap these lines as they and others in this file are all <120. :thinking: